### PR TITLE
Doi bugs

### PR DESCRIPTION
**Pull request by sri2k1us**
- core-1492 Fix errors when updating DOI request status.

- core-1493 Fix error that when trying to view metadata for DOI request.

### CODE REVIEW

File: src/components/utils/UpdateRequestDialog.js [hunk 1]
```diff
@@ -46,32 +46,40 @@ export default function UpdateRequestDialog(props) {
...
-            enabled: !!requestId && open && requestType === RequestType.DOI,
-            onSuccess: (data) => {
-                setRequestDetails(data);
-            },
-        });
+    const { isFetching: isDOIRequestFetching } = useQuery({
+        queryKey: [REQUEST_DETAILS_QUERY_KEY, { id: requestId }],
+        queryFn: () => adminGetRequestDetails({ id: requestId }),
+        enabled: !!requestId && open && requestType === RequestType.DOI,
+        error: setDoiRequestFetchError,
```
Comment (ash3rz) [inline src/components/utils/UpdateRequestDialog.js:62]: I think this key in the `useQuery` config is `onError`

File: src/components/utils/UpdateRequestDialog.js [hunk 2]
```diff
@@ -46,32 +46,40 @@ export default function UpdateRequestDialog(props) {
...
-            enabled: !!requestId && open && requestType === RequestType.TOOL,
-            onSuccess: (resp) => {
-                setRequestDetails(resp);
-            },
-        });
+    const { isFetching: isToolRequestFetching } = useQuery({
+        queryKey: [ADMIN_TOOL_REQUEST_DETAILS_QUERY_KEY, { id: requestId }],
+        queryFn: () => getAdminToolRequestDetails({ id: requestId }),
+        enabled: !!requestId && open && requestType === RequestType.TOOL,
+        error: setToolRequestFetchError,
```
Comment (ash3rz) [inline src/components/utils/UpdateRequestDialog.js:75]: Same comment - I think this key in the `useQuery` config is `onError`

**Review by ash3rz — APPROVED** [review]
Think I caught 2 typos, but LGTM :shipit:

File: src/components/utils/UpdateRequestDialog.js (hunk 1 above)
Comment (psarando) [reply src/components/utils/UpdateRequestDialog.js:62]: I also think this key should be `onError` 👍

File: src/components/utils/UpdateRequestDialog.js (hunk 2 above)
Comment (psarando) [reply src/components/utils/UpdateRequestDialog.js:75]: I also think this key should be `onError` 👍

**Review by psarando — APPROVED** [review]
LGTM, except for the `onError` typos 👍

File: src/components/utils/UpdateRequestDialog.js (hunk 1 above)
Comment (sri2k1us) [reply src/components/utils/UpdateRequestDialog.js:62]: Oops. Good Catch.

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**